docs/source/examples/demo_metrics_correlation.ipynb

```diff
@@ -0,0 +1,414 @@
+   "outputs": [
+    {
+     "data": {
+      "text/plain": "{'function_name': 'METRICS_CORRELATION',\n 'output': {'inference_results': [{'event_window': [2.0, 94.0],\n    'event_pattern': [0.16287264,\n     0.17291106,\n     0.16507792,\n     0.1442258,\n     0.11990739,\n     0.09729371,\n     0.080975406,\n     0.07199166,\n     0.06942996,\n     0.07270404,\n     0.08256501,\n     0.100012146,\n     0.12396885,\n     0.14978407,\n     0.17256345,\n     0.18076059,\n     0.17145823,\n     0.14817578,\n     0.12038868,\n     0.09797779,\n     0.089608796,\n     0.09555778,\n     0.11185065,\n     0.13050617,\n     0.14417148,\n     0.14867567,\n     0.14488967,\n     0.13812506,\n     0.13548654,\n     0.14443183,\n     0.16681649,\n     0.20366035,\n     0.25175166,\n     0.31876424,\n     0.38990238,\n     0.4533083,\n     0.5011735,\n     0.5278964,\n     0.5301469,\n     0.5062882,\n     0.45027423,\n     0.44566932,\n     0.4209668,\n     0.38853073,\n     0.35869414,\n     0.3340214,\n     0.31001407,\n     0.28004804,\n     0.24047177,\n     0.19322383,\n     0.15334728,\n     0.11692942,\n     0.08834248,\n     0.068126686,\n     0.05413547,\n     0.043827415,\n     0.03604572,\n     0.031724308,\n     0.03348354,\n     0.044769436,\n     0.06946566,\n     0.10541002,\n     0.14753905,\n     0.18694052,\n     0.2131184,\n     0.21818678,\n     0.2007543,\n     0.16648845,\n     0.13061307,\n     0.099168815,\n     0.07933296,\n     0.07126682,\n     0.07225323,\n     0.07784612,\n     0.08396258,\n     0.09038867,\n     0.09414727,\n     0.09750885,\n     0.10493818,\n     0.12241074,\n     0.15522112,\n     0.20462617,\n     0.26510048,\n     0.32717478,\n     0.3735336,\n     0.3883701,\n     0.36791247,\n     0.32734427,\n     0.27416283,\n     0.22305378,\n     0.18449455,\n     0.16176924,\n     0.15200219,\n     0.14995821,\n     0.15202914,\n     0.15801251,\n     0.17027171,\n     0.19150275,\n     0.21942066],\n    'suspected_metrics': [2,\n     3,\n     6,\n     10,\n     15,\n     18,\n     19,\n     20,\n     21,\n     24,\n     27,\n     29,\n     30,\n     32,\n     34,\n     35]}]}}"
+     },
+     "execution_count": 13,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "results = ml_client.execute(\n",
+    "    algorithm_name = \"METRICS_CORRELATION\",\n",
+    "    input_json = input_json\n",
+    ")\n",
+    "results\n"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Step 4: Result visualization\n"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 33,
+   "outputs": [
+    {
+     "data": {
+      "text/plain": "Text(0.5, 1.0, 'All metrics')"
+     },
+     "execution_count": 33,
+     "metadata": {},
+     "output_type": "execute_result"
+    },
+    {
+     "data": {
+      "text/plain": "<Figure size 640x480 with 1 Axes>",
```

Comment:
Can you share any notebook url from your repo which has the same output? It's not quite possible to review from here.

Reply:
Can you see it from [here](https://github.com/opensearch-project/opensearch-py-ml/blob/f1886bad74b73bed9cf54eb2493b3dcf781a2e1f/docs/source/examples/demo_metrics_correlation.ipynb)?

Reply:
Can you see it?

Reply:
Yeah, I can see it. Sorry I forgot to reply. @greaa-aws will also review your PR and comment here as well.